docker-ci.sh

```diff
@@ -195,6 +195,13 @@ function run_test_scss {
     npm i && npm run build && npm run test
 }
 
+function run_test_license {
+    npm i
+    npm run build
+
+    npx gulp check-license-notices
```

Comment:
This task is called twice. See:
https://devextreme-ci.devexpress.com/DevExpress/DevExtreme/34035/51

Reply:
I've updated PR and then we call 'check-license-notices' task only in 'test_license' task